.yamllint.yaml

```diff
@@ -0,0 +1,7 @@
+extends: default
+
+rules:
+  line-length:
+    level: warning
+
+  indentation: disable
```

Comment:
This default rule wants arrays to be indented as well. I've turned this off since all the yaml files in this repo do not indent arrays. If desired, I can re-enable and update all the files.